assets/js/blocks/mini-cart/style.scss

```
@@ -42,23 +42,36 @@
 .wc-block-mini-cart__drawer {
 	font-size: 1rem;
 
+	.components-modal__content .components-modal__header .components-button {
+		top: 50%;
+		padding: unset;
+		margin: unset;
+	}
+
+	.wc-block-components-notices {
+		padding: 0 16px;
+		margin-bottom: unset;
```

Here we set some extra margin to the notice wrapper:
```suggestion
		margin: #{$gap} #{$gap} -#{$gap};
```